service/user/user.go

```diff
@@ -99,22 +71,25 @@ func convertUsers(rcd [][]string) (*[]model.Users, int, error) {
 			u = append(u, *data)
 		}
 	}
-	fmt.Println("Files has invalid records - #total: ", invalidRecords)
+	if invalidRecords > 0 {
+		fmt.Println("Files has invalid records - #total: ", invalidRecords)
+	}
 	return &u, invalidRecords, nil
 }
 
-func parseUserRecord(r []string) (*model.Users, error) {
-	id, err := strconv.Atoi(r[0])
+func getUser(id int, c CSV) (*model.Users, error) {
+	u := &model.Users{}
+	rcd, err := c.GetData()
 	if err != nil {
-		return nil, errors.New("invalid record")
+		log.Print(err.Error())
+		return u, err
 	}
-	//status, _ := strconv.ParseBool(r[4])
-	u := model.Users{
-		ID:     id,
-		Name:   r[1],
-		Email:  r[2],
-		Gender: r[3],
-		Status: r[4],
+
+	for _, r := range rcd {
+		if r[0] == strconv.Itoa(id) {
+			u, _ = parseUserRecord(r)
```

Comment:
Handle the error

Reply:
Hi! This is a integer to string, that doesn't returns an error.